airbyte-server/src/main/java/io/airbyte/server/ConfigDumpImporter.java

```diff
@@ -484,25 +485,25 @@ public void importIntoWorkspace(String targetVersion, UUID workspaceId, File arc
                   configRepository.getDestinationConnection(standardSync.getDestinationId()) == null) {
                 return;
               }
-              for (UUID operationId : standardSync.getOperationIds()) {
+              for (final UUID operationId : standardSync.getOperationIds()) {
                 if (configRepository.getStandardSyncOperation(operationId) == null) {
                   return;
                 }
               }
-            } catch (ConfigNotFoundException e) {
+            } catch (final ConfigNotFoundException e) {
               return;
             }
             configRepository.writeStandardSync(standardSync);
           });
     }
   }
 
-  protected <T> void importSourceDefinitionIntoWorkspace(Stream<T> configs)
+  protected <T> void importSourceDefinitionIntoWorkspace(final Stream<T> configs)
```

Comment:
@ChristopheDuong i think this just a typo where this class is supposed to be pulling standard source definitions but instead pulls standard sources.

Reply:
the method is named (confusingly) `listStandardSources` but returned a `list<StandardSourceDefinitions>` though:

https://github.com/airbytehq/airbyte/blob/da3677e821afc8a385c4c937692a9b039425b8fb/airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/ConfigRepository.java#L152

Reply:
Shouldn't it be removed completely now that you added `listStandardSourceDefinitions`?

Reply:
whacky!